crowbar_framework/app/models/backup.rb

```diff
@@ -23,6 +23,9 @@ class Backup < ActiveRecord::Base
   before_validation :save_or_create_archive, on: :create
   after_validation :delete_archive, unless: -> { errors.empty? }
   after_destroy :delete_archive
+  after_create do
+    data.rmtree
```

Comment:
when I checked, my `/tmp` folder has not been cleaned up. when exactly does this garbage collection happen?